exponent_server_sdk/__init__.py

```diff
@@ -414,6 +428,9 @@ def publish_multiple(self, push_messages):
         """
         push_tickets = []
         for start in itertools.count(0, self.max_message_count):
+            # Todo : Check if len(push_message.to) check is required here as well
```

Comment:
From reading the [docs](https://docs.expo.dev/push-notifications/sending-notifications/) it looks like it's 100 total (including the different ones, or same one with multiple "to's"